weblab/repocache/migrations/0012_split_cache.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-11-20 11:56
```

Comment:
does this suggests it was auto generated? but we are adding manually?

Reply:
There's a Django manage.py command that creates the file template for you, filling in the migration dependencies.

Reply:
thanks.. I'm interested why we need the reverse case "combine_caches" do you have to support going back?

Reply:
maybe we should call the file _split_and_combine_ cache

Reply:
yeah, mainly to support developers - so you can try out the branch, and roll back the DB if switching to another branch.

Reply:
Regarding the rename query, I think most migrations I've seen have been named more like the forward direction, but it's not a major point.